stepactions/skip-trusted-artifact-operations/skip-trusted-artifact-operations.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: StepAction
+metadata:
+  name: skip-trusted-artifact-operations
+spec:
+  description: >-
+    This stepaction creates the file .skip-trusted-artifacts if the param ociStorage is
+    "empty".
+  image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+  params:
+    - name: ociStorage
+      type: string
+      default: "empty"
+    - name: subdirectory
```

Comment:
Is there any reason to not include this in dataDir? And save needing this parameter

Reply:
fixed